adapters/huaweiads/huaweiads.go

```diff
@@ -982,8 +1062,9 @@ func (a *adapter) handleHuaweiAdsContent(adType int32, content *content, bidType
 // extractAdmBanner: banner ad
 func (a *adapter) extractAdmBanner(adType int32, content *content, bidType openrtb_ext.BidType, imp openrtb2.Imp) (adm string,
 	adWidth int64, adHeight int64, err error) {
-	if adType != banner {
-		return "", 0, 0, errors.New("extractAdmBanner: huaweiads response is not a banner ad")
+	// support openrtb: banner  <=> huawei adtype: banner, interstitial
+	if adType != banner && adType != interstitial {
+		return "", 0, 0, errors.New("extractAdmBanner: huaweiads response adtype doesn't match")
```

Comment:
I'd recommend not using the function name here in the error message.

Reply:
corrected，thank you